code/datums/outfit.dm

```diff
@@ -67,7 +67,8 @@
 
 	if(accessory)
 		var/obj/item/clothing/under/U = H.w_uniform
-		U.attach_accessory(new accessory(H))
+		if(U)
+			U.attach_accessory(new accessory(H))
 
```

Comment:
else
    WARNING("Unable to equip accessory [accessory] in outfit [name]. No uniform present!")